umodbus/serial.py

```diff
@@ -22,13 +19,10 @@
 from .common import ModbusException
 from .modbus import Modbus
 
-# typing not natively supported on MicroPython
-from .typing import List, Optional, Union
-
 
 class ModbusRTU(Modbus):
     """
-    Modbus RTU client class
+    Modbus RTU server class
```

Comment:
```suggestion
    Modbus RTU client class
```

This is a client, serving data to be requested by a host

Reply:
I think this is a Modbus master? Master == client, slave == server since the slave/server has data that a master/client can request